front_end/src/app/app.component.ts

```diff
@@ -51,7 +51,9 @@ export class AppComponent implements OnInit {
 
   ngOnInit() {
     // Start the signalr connection up!
-    this.channelService.start();
+    if (process.env.DISABLE_WEBSOCKETS != 'true') {
+      this.channelService.start();
+    }
 
```

Comment:
this doesnt have anything to do with this story, but allows us to run locally and hit int backend